erpnext/regional/united_arab_emirates/utils.py

```diff
@@ -20,6 +16,27 @@ def update_itemised_tax_data(doc):
 
 	itemised_tax = get_itemised_tax(doc.taxes)
 
+	def determine_if_export(doc):
+		if doc.doctype != "Sales Invoice":
+			return False
+
+		if not doc.customer_address:
+			frappe.msgprint(
+				_("Please set Customer Address to determine if the transaction is an export."),
+				alert=True,
+			)
```

Comment:
This may be an extra alert for all b2c transactions. Hence should be avoided.

Reply:
Added a condition, the message will be show only if no tax is applied.